release.py

```diff
@@ -421,9 +421,8 @@ def cf_hash(opts, pargs):
     if len(pargs) > 2:
         parser.error("Improper usage. Too many arguments!")
 
-    # Prevent release and pre_release both being true
     if opts.release and opts.pre_release:
-        parser.error("Cannot release and pre-release at the same time.")
+        parser.error("Please re-run the command specifying either release or pre_release.")
```

Comment:
This is missing the error. It only has what to do about it. We want the mesage to say what went wrong then how to fix it.